src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -71,20 +95,97 @@ export const authOptions: AuthOptions = {
   callbacks: {
     async jwt({ token, account, profile, trigger }) {
       if (profile) {
-        token.locale = profile.locale;
-        token.twoFactorAuthentication = profile.twoFactorAuthentication;
-        token.metricsEnabled = profile.metricsEnabled;
-        token.avatar = profile.avatar;
-        token.avatarDefault = profile.avatarDefault;
+        token.fxa = {
```

Comment:
I see that we are putting all the signup code here... which reminded me - we still need a place to put all the log out code?

Reply:
I think we're good - logout should just be a call to Next-Auth's [`signOut` function](https://next-auth.js.org/getting-started/client#signout), I believe.